src/main/java/nl/tudelft/pixelperfect/event/parameter/EventParameterCollection.java

```diff
@@ -86,9 +86,15 @@ public String toDebugString() {
       sb.append("None");
     } else {
       for (EventParameter param : map.values()) {
-        sb.append(param.getNumberValue());
-        sb.append(',');
-      }      
+        if (!param.getValue().equals("") && !param.getValue().equals("Generic")) {
+          if (sb.length() == 0) {
+            sb.append(param.getValue());
+          } else {
+            sb.append(", ");
```

Comment:
I'm getting a checkstyle error, claiming it should be a 'char.'  However, this is obviously wrong, since you want the space.  I'd add an  @suppress.